src/AutoRest.CSharp/Mgmt/Models/MgmtClientOperation.cs

```diff
@@ -134,14 +134,23 @@ public MethodSignature MethodSignature
         private FormattableString BuildDescription()
         {
             var pathInformation = _operations.Select(operation =>
-                (FormattableString)$@"<item>
+            {
+                string resourceTypeItem = string.Empty;
+                if (operation.Resource != null)
+                    resourceTypeItem = $@"
+<item>
+<term>Resource Type</term>
+<description>{operation.Resource.ResourceType}</description>
```

Comment:
if you want to add a cref of a type, you could just write:
FormattableString s = $"{operation.Resource:C}";
when writer writes this, we will get:
<see cref="Namespace.Name"/>